keywords/gesturerecognition.md

```diff
@@ -0,0 +1,10 @@
+Human-Machine social [interaction](../main/glossary/.md#computer-human_interaction) usually relies on verbal or written communication, this is not the case for gesture recognition. Gesture Recognition is an area of computation science which detects and interprets human body movement. Much of this is done using [machine learning](../main/glossary/.md#machinelearning) algorithims[^kobylarz01wiki]. 
```

Comment:
As stated in the instructions, you need to add a heading 2 title for this entry:

```
## Gesture recognition
```